modules/cache-common/src/test/java/org/opensearch/cache/common/tier/MockDiskCache.java

```diff
@@ -128,6 +128,10 @@ public void close() {
 
     }
 
+    long getMaxSize() {
+        return maxSize;
+    }
```

Comment:
I am wondering if we really need this method, given it is not there in `ICache` interface

Reply:
@jainankitk As discussed below, we need this to be used only within the UTs.